stix_shifter_modules/tanium/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,1058 @@
+        },
+        {
+            "key": "x-oca-event.ip_refs",
+            "object": "event",
+            "references": [
+                "ip"
+            ]
+        },
+        {
+            "key": "x-oca-asset.ip_refs",
+            "object": "target-asset",
+            "references": [
+                "target-ip"
+            ]
+        }
+    ],
+    "computerName": [
+        {
+            "key": "x-oca-asset.hostname",
+            "object": "target-asset"
+        },
+        {
+            "key": "x-oca-event.host_ref",
+            "object": "event",
+            "references": "target-asset"
+        }
+    ],
+    "state": {
+        "key": "x-oca-event.outcome",
+        "object": "event"
+    },
+    "severity": [
+        {
+            "key": "x-ibm-finding.severity",
+            "object": "alert",
+            "transformer": "ConvertTextSeverityToNumberValue"
+        },
+        {
+            "key": "x-oca-event.severity",
+            "object": "event",
+            "transformer": "ConvertTextSeverityToNumberValue"
+        }
+    ],
+    "details": 
+    {
+        "match": 
+        {
+            "hash": 
+            {
+                "key": "x-ibm-finding.x_match_hash",
+                "object": "alert"
+            },
+            "type": 
+            {
+                "key": "x-ibm-finding.x_match_type",
+                "object": "alert"
+            },
+            "source": 
+            {
+                "key": "x-ibm-finding.x_match_source",
+                "object": "alert"
+            },
+            "version": 
+            {
+                "key": "x-ibm-finding.x_match_version",
+                "object": "alert"
+            },
+            "contexts": 
+            {
+                "file": 
+                {
+                    "uniqueEventId": 
+                    {
+                        "key": "x-ibm-finding.x_match_unique_event_id",
+                        "object": "alert"
+                    }
+                },
+                "event":
+                {
+                    "fileCreate": 
+                    {
+                        "path": 
+                        {
+                            "key": "x-ibm-finding.x_match_path",
+                            "object": "alert"
+                        },
+                        "timestampMs": 
+                        {
+                            "key": "x-ibm-finding.x_match_timestamp",
+                            "object": "alert"
+                        }
+                    }
+                }
+            },
+            "properties": 
+            {
+                "pid": [
+                    {
+                        "key": "process.pid",
+                        "object": "match-process"
+                    },
+                    {
+                        "key": "x-ibm-finding.x_match_process_ref",
+                        "object": "alert",
+                        "references": "match-process"
```

Comment:
you should map this to "x-oca-event.process_ref" instead of "x-ibm-finding.x_match_process_ref"